userguide/storage.rst

```diff
@@ -2091,37 +2089,35 @@ different :guilabel:`Transport` options:
    +===========================+===========+================+=================================================================================================================+
    | Name                      | ALL       | string         | Enter a descriptive :guilabel:`Name` for the replication.                                                       |
```

Comment:
Use upper/lower "All" to differentiate the non-literal "All" from the specific types in upper case.